### ISSUE TITLE
less than 3 moves positions

### ISSUE THREAD

**Issue by Disservin** [MakotoE/guess-the-eval#8]
I didnt take a look at the code but I just had this position
`5r1k/3Q4/3p3p/p1pP3P/3pB1R1/1P1Pb3/6K1/5q2 w - - 2 41`
White has 2 moves in this position and no matter which one you take, you will still have made one of Stockfish's top 3 moves.
This is kinda dumb in my opinion, since you get a forced multiplier.

**Comment by MakotoE** [comment]
But one of them awards x3 and the other is less than 3, right?

**Comment by Disservin** [comment]
Yes, just that guess the eval maybe shouldnt have these kinds of positions with only 1-3 moves.

**Comment by MakotoE** [comment]
I'm in agreement that positions with only 1-2 valid moves should not be included. What about 3 moves? I think there is some value in including those.
